src/database/seeders/20190910075430-test-user.js

```diff
@@ -9,10 +9,22 @@ export function up(queryInterface, Sequelize) {
     username: 'johndoe',
     email: 'technitesdev@gmail.com',
```

Comment:
I have a seed that makes this specific user as superadmin, since this is a functional email